parser.py

```diff
@@ -421,7 +159,7 @@ def goplist(self, *args):
         if len(args) == 1:
             # <uop>
             # <term>
-            return NamedList("goplist", flatten(args))
+            return ParsedList(Lists.GOPLIST, flatten(args))
```

Comment:
Felt that enums were a better method to represent these various list structures. By the time they appear in the Transformer, the type of list is more or less just for bookkeeping for the grammar, and has no real consequence

Reply:
🙏 💯